GameOfRunes/GameOfRunes/GameEngine/Systems/ManaSystem.swift

```diff
@@ -38,7 +38,7 @@ class ManaSystem: GKComponentSystem<ManaComponent>, System {
 }
 
 /** Extension for the ManaSystem to deal with DroppedMana-related events and logic. */
-extension ManaSystem: DroppedManaResponder {
+extension ManaSystem {
     /** Function called whenever a monster is killed and mana can be dropped. */
     func dropMana(at enemyEntity: GKEntity) {
```

Comment:
I think can move this to GameEngine's extension

Reply:
Think can put it here for now, with the meta-data in sprint 2, we will then see how